framework/base/Module.php

```diff
@@ -729,4 +729,21 @@ public function get($id, $throwException = true)
         }
         return $component;
     }
+
+    /**
+     * @inheritdoc
+     * @param string $id
+     * @param bool $checkInstance
+     * @return bool
+     * Since version 2.0.13, if a component isn't defined in the module, it will be looked up in the parent module.
+     * The parent module may be the application.
+     */
+    public function has($id, $checkInstance = false)
```

Comment:
How about the following variant? Looks a bit more readable to me...

```php
if (parent::has($id, $checkInstance)) {
    return true;
}

if (isset($this->module)) {
    return $this->module->has($id, $checkInstance);
}

return false;
```

Reply:
I've changed it to a one liner that feels more readable; i'm not a fan of using `if... return true;`.

Reply:
Well, my variant is close in reading to description. Yours is not and it duplicates a single condition multiple times.

Reply:
OK. New one looks fine.